### ISSUE TITLE
Correcting wording and details for interface text. (second report)

### ISSUE THREAD

**Issue by Calyp** [freedombox/Plinth#380]
Item No 108:
"Direct connection to the internet."

>Internet is always written with capital I.
--->"Direct connection to the Internet."

Item No 109:
"Behind NAT, this means that dynamic DNS service will poll the "IP check URL" for changes (we need the "IP check URL" for this reason - otherwise we will not detect IP changes). It may take up to %(timer)s minutes until we update your DNS entry in case of WAN IP change."

>Who is "we"? This sounds funny. Also odd punctuation and grammar, suggested text:
--> "Behind NAT. This means that the dynamic DNS service will poll the "IP check URL" for changes (the "IP check URL" entry is needed for for this - otherwise IP changes will not be detected). It may take up to %(timer)s minutes until your DNS entry is updated, in case the WAN IP changes.

Item No 112"
"Firewall is a network security system that controls the incoming and outgoing network traffic on your %(box_name)s. Keeping a firewall enabled and properly configured reduces risk of security threat from the Internet."

>Redundant wording. ("network" twice) I suggest changing this to:
--> A firewall is a network security system that controls the incoming and outgoing traffic on your %(box_name)s. ... "  
or
?--> " A firewall is a security system that controls the incoming and outgoing network traffic on your %(box_name)s. ... "

Item No 113:
"The following is the current status:"

>This seems excessive, colon already indicates "the following". Suggesting reduction to:
--> "Current status:"
Alternatively (better sounding but still excessive?)
?--> "Current status is as follows:"

Item No 122:
"The operation of the firewall is automatic. When you enable a service it is automatically permitted in the firewall and you disable a service is automatically disabled in the firewall."

>Incomplete wording in last phrase, and redundant wording. Suggested change:
--> "The firewall is operated automatically. When you enable a service it is also permitted in the firewall and when you disable a service it is also disabled in the firewall."

Item No 133:
"%(box_name)s setup is now complete.  To make your %(box_name)s functional, you need some applications.  Applications will be installed the first time you access them." 

>Redundant wording. Suggested change:
--> "%(box_name)s setup is now complete.  To make your %(box_name)s functional, you need some applications.  They will be installed the first time you access them."

Item No 136:
"You should check the network setup and modify it if necessary. Do not forget to change the default Wi-Fi passwords!"  

>Should it really be "Wi-Fi passwords" in plural? 

Item No 144:
"FreedomBox is a community project to develop, design and promote personal servers running free software for private, personal communications.  It is a networking appliance designed to allow interfacing with the rest of the Internet under conditions of protected privacy and data security.  It hosts applications such as blog, wiki, website, social network, email, web proxy and a Tor relay on a device that can replace your Wi-Fi router so that your data stays with you."

>The three sentences are separated by double spaces. 
It could use some punctuation and grammar correction.
FreedomBox is a software-stack, not an appliance. 
We might want to emphasize that the data is personal or sensitive. Suggested change:
--> "FreedomBox is a community project to develop, design and promote personal servers running free software for private and personal communication. It is a networking appliance designed to allow interfacing with the rest of the Internet, under conditions of protected privacy and high data security. It hosts applications such as blog, wiki, website, social network, email, web proxy and a Tor relay, on a device that can replace your Wi-Fi router, so that your personal data always stays under your control."

Item No 145:
"We live in a world where our use of the network is mediated by those who often do not have our best interests at heart.  By building software that does not rely on a central service, we can regain control and privacy.  By keeping our data in our homes, we gain useful legal protections over it.  By giving back power to the users over their networks and machines, we are returning the Internet to its intended peer-to-peer architecture."

>Again, this text has double space separation of sentences. Fix.

Item No 147:
"For more information about the FreedomBox project, see the <a href="https://wiki.debian.org/FreedomBox">FreedomBox Wiki</a>."

>For consistency, "Wiki" should not be typed with capital "w"
--> "For more information about the FreedomBox project, see the <a href="https://wiki.debian.org/FreedomBox">FreedomBox wiki</a>."

Item No 151:
"The <a href="%(manual_url)s">FreedomBox Manual</a> is the best place to start for information regarding %(box_name)s."

>For consistency, "Manual" should not be typed with capital "m"
--> "The <a href="%(manual_url)s">FreedomBox manual</a> is the best place to start for information regarding %(box_name)s."

Item No 155, 156, 157 :
155 "Wiki & Blog (Ikiwiki)"
156 "Ikiwiki wikis and blogs"
157 "Enable Ikiwiki"

>ikiwiki is a brand name, and as such we should use the stylized form with no capitalization for consistency, also "Blog" in "Wiki & Blog" should not use capital "B" or "&" :
155 --> "Wiki and blog (ikiwiki)"
156 --> "ikiwiki wikis and blogs"
157 --> "Enable ikiwiki"

Item No 163:
"Delete Wiki/Blog <em>%(name)s</em>"

>For consistency, we should not write "Wiki/Blog with capital letter, and not use slash for or/and.
Suggested text: 
-->"Delete wiki or blog <em>%(name)s</em>"

Item No 164:
"This action will remove all the posts, pages and comments including revision history.  Delete this wiki/blog permanently?"

>We should not use slash for or/and.
Suggested text: 
-->"This action will remove all the posts, pages and comments including revision history.  Delete this wiki or blog permanently?"

Item No 168:
"Create a Wiki or Blog"
>For consistency, we should not write "Wiki or Blog" with capital letter:
-->"Create a wiki or blog"

Item No 173:
"Wiki & Blog"

>For consistency, we should not write "Blog" with capital letter or use "&":
-->"Wiki and blog"

Item 174:
"Manage Wikis and Blogs"

>For consistency, we should not write "Wiki and Blog" with capital letter:
"Manage wikis and blogs"

Item No 175 and 182:
175 "Create Wiki/Blog"
182 "Delete Wiki/Blog"

>For consistency, we should not write "Wiki/Blog with capital letter, and not use slash for or/and.
Suggested text: 
175 --> "Create wiki or blog"
182 --> "Delete wiki or blog"

Item No 187 and 188
187 "Certificate Status"
188 "Website Security"

>For consistency, we should not write "Status" and "Security" with capital letter
187 --> "Certificate status"
188 --> "Website security"

**Comment by SunilMohanAdapa** [comment]
I fixed most of the items as mentioned except the following comments:

> "Behind NAT, this means that dynamic DNS ...

Slightly rephrased the last sentence.

> "You should check the network setup ...

There could be multiple Wi-Fi interfaces with multiple passwords set.  It is correct to say "passwords".  Although we might still think of it.

> FreedomBox is a community project..

In "Private, personal communications" it is correct to use a comma as the adjectives are coordinate adjectives.  Updated the punctuation in the last statement.  However, I kept the phrase "data stays with you" because I believe it conveys more information than "data always stays under your control".

> We live in a world where our use of the network ...

Double spaces are fine for separating sentences because this is a style.  Although in writing we don't do this, in computers it is common as there is no easy way for machines to distinguish fullstops used after abbreviations from fullstops used after sentences.  Editors like Emacs realize double spaces differently and help with breaking long lines more intelligently.

Of course, it would have been nice to have consistency.  But in this case, it is not too much of a problem and perhaps we can keep it.

> the FreedomBox Wiki
> The FreedomBox Manual

Both uses are as proper nouns. The title casing is okay.

>  "Delete Wiki/Blog %(name)s"
> "Create a Wiki or Blog"
> "Manage Wikis and Blogs"
> 175 "Create Wiki/Blog"
> 182 "Delete Wiki/Blog"
> 187 "Certificate Status"
> 188 "Website Security"

Fixed "/" to "or" or "and".  Kept the casing as these are the titles of a page, labels of buttons or heading of a table column.
